txauthor.go

```diff
@@ -64,6 +69,31 @@ func (tx *TxAuthor) SendDestination(atIndex int) *TransactionDestination {
 	return &tx.destinations[atIndex]
 }
 
+func (tx *TxAuthor) AddChangeDestination(address string, atomAmount int64) int {
```

Comment:
- Why does change destination have an amount parameter?
- How will the user know the change amount without knowing the fee?
- What happens if the change amount is lower than the actual change and will this result in paying a higher fee?